website/source/docs/job-specification/ephemeral_disk.html.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: "docs"
+page_title: "ephemeral_disk Stanza - Job Specification"
+sidebar_current: "docs-job-specification-ephemeral_disk"
+description: |-
+  The "ephemeral_disk" stanza instructs Nomad to utilize an ephemeral disk
+  instead of a hard disk requirement, and can also enable sticky volumes and
```

Comment:
The "ephemeral_disk" stanza describes the ephemeral disk requirements of the task group. Ephemeral disks can be marked as sticky and support live data migrations